app/src/components/Dashboard/SideNavbar.tsx

```diff
@@ -0,0 +1,203 @@
+      <Text
+        fontSize="lg"
+        fontWeight="semibold"
+        style={{ textAlign: 'left', verticalAlign: 'center' }}
+      >
+        {text}
+      </Text>
+    </Box>
+  );
+}
+
+export function SideNavbar({
+  activePage,
+  onSelectPage,
+}: {
+  activePage: TPages;
+  onSelectPage: (page: TPages) => void;
+}) {
+  const { isPremium, setPremium } = useGoldContext();
+  const {
+    isOpen: isPremiumModalOpen,
+    onClose: onClosePremiumModal,
+    onOpen: onOpenPremiumModal,
+  } = useDisclosure();
+  const {
+    isOpen: isEmailModalOpen,
+    onClose: onCloseEmailModal,
+    onOpen: onOpenEmailModal,
+  } = useDisclosure();
+
+  return (
+    <div
+      style={{
+        height: '100vh',
+        borderRight: '1px solid gray',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+      }}
+    >
+      <div>
+        <div>
+          <div style={{ padding: '18px' }}>
+            <img
+              src={LogoWithText}
+              alt="Left on Read"
+              style={{ height: '100%' }}
+            />
+          </div>
+        </div>
+        <div style={{ marginTop: '25%' }}>
+          <Stack direction="column" alignItems="flex-start">
+            {Pages.filter((page) => page !== 'Settings').map((page) => {
+              const icon = BsLightningCharge;
+
+              // if (page === 'Analytics') {
+              //   icon = FiBarChart;
+              // } else if (page === 'Reports') {
+              //   icon = FiClipboard;
+              // }
+
+              return (
+                <SidebarMainLink
+                  key={page}
+                  text={page}
+                  icon={icon}
+                  isActive={activePage === page}
+                  onSelect={() => {
+                    onSelectPage(page);
+                  }}
+                />
+              );
+            })}
+          </Stack>
+        </div>
+      </div>
+      <div
+        style={{
+          backgroundColor: defaultTheme.colors.gray['200'],
+          padding: '26px',
+        }}
+      >
+        <Stack
+          direction="column"
+          style={{ alignItems: 'flex-start', marginTop: '22px' }}
+          spacing="12px"
+        >
+          {!isPremium && (
+            <Button
+              variant="link"
+              fontSize="md"
+              colorScheme="yellow"
+              onClick={onOpenPremiumModal}
+              fontWeight="semibold"
+            >
+              Unlock Gold
+            </Button>
+          )}
+          <Button
+            variant="link"
+            fontSize="md"
+            onClick={() => {
+              onSelectPage('Settings');
+            }}
+          >
+            Settings
+          </Button>
+          <Button
+            variant="link"
+            fontSize="md"
+            onClick={() => {
+              onOpenEmailModal();
+            }}
+          >
+            Feedback
```

Comment:
```suggestion
            Contact Support
```

I feel like if users have a bug, they'll wanna see how to contact support. Feedback less likely